template/src/module.ts

```diff
@@ -0,0 +1,23 @@
+import {RouteConfig} from 'vue-router';
+
+const homeComponent = () => import('./components/home').then(({HomeComponent}) => HomeComponent);
+const aboutComponent = () => import('./components/about').then(({AboutComponent}) => AboutComponent);
+const listComponent = () => import('./components/list').then(({ListComponent}) => ListComponent);
+
+
+export function createRoutes(prefix: string = ''): RouteConfig[] {
+  return [
+    {
+      path: prefix + '/',
+      component: homeComponent,
+    },
+    {
+      path: prefix + '/about',
+      component: aboutComponent,
+    },
+    {
+      path: prefix + '/list',
+      component: listComponent,
+    }
+  ];
+}
```

Comment:
If you move this to a `routes.ts` file, I think it would be better for the `module.ts` to import from `routes.ts`. This way it keeps functions and constants in the files that describe them.

Reply:
as I wrote on the previous comment, it conflicts the module mechanism..

Reply:
I understand, I meant the naming of the file feels wrong as it is a `createRoutes` function. Splitting the files and naming them accordingly makes sense even if they only import/export a single function.